backend/common/src/main/java/ai/verta/modeldb/common/CommonDBUtil.java

```diff
@@ -90,13 +90,18 @@ protected void releaseLiquibaseLock(DatabaseConfig config)
 
       try (var stmt = jdbcCon.createStatement()) {
 
-        var sql = "SELECT * FROM database_change_log_lock WHERE ID = 1";
-        ResultSet rs = stmt.executeQuery(sql);
+        ResultSet rs;
+        try {
+          var sql = "SELECT * FROM database_change_log_lock WHERE ID = 1";
+          rs = stmt.executeQuery(sql);
+        } catch (Exception e) {
+          rs = null;
```

Comment:
Add a log message about this table being missing. 

Maybe instead of setting to `null`, we should `return` out of this method? 

The rest of this method doesn't make sense without this ResultSet.

Reply:
Or maybe UAC should perform this check? This code is not needed to for any other projects.